### DISCUSSION TITLE
Different travel rout for non members and members

### DISCUSSION THREAD

**Discussion by ghost** [DiscordDungeons/Bugs#964] [Ideas]
some travel location have two rout, membership and non membership. what's weird is that non membership constantly get suggested to use the membership rout. this cause huge annoyance got non membership. i suggest make drpg able to check if a member is non membership or membership then use the correct rout (ie membership go membership rout, non membership go non membership rout)

**Comment by ichlaffterlalu** [comment]
To make it more clear, I will give an example:
Travel from Mournstead (Illeria kingdom) to Hurrenberg (Brimstone kingdom)

Non-members cannot travel directly to Hurrenberg via shortest path:
_Mournstead_ -> Rivermouth -> **Island of Desolation** -> Brimstone -> Peppershore -> _Hurrenberg_
Island of Desolation is on Liriania kingdom which is membership-only.

While actually, non-members *can* travel to Hurrenberg via another path, but it is not the shortest one:
_Mournstead_ -> Rivermouth -> **Sariila** -> **Glimmerhelm** -> Brimstone -> Peppershore -> _Hurrenberg_

What we want is `#!travel` offers shortest **possible** path, so we didn't need to manually travel multiple times.
This also applies to level or quest-restricted places.
